src/controllers/TalksController.php

```diff
@@ -2,16 +2,18 @@
 
 class TalksController extends ApiController {
     public function handle(Request $request, $db) {
```

Comment:
We aren't using the `handle` method any more as we route directly in `2.1.json`, so no need to change it. At some point, we should remove the abstract method from `ApiController`…